### PR TITLE
Update clang-format and buildifier.

### PR DESCRIPTION
**Pull request by PiotrSikora**
This prevents CI from dowloading clang-format-8 all the time.

Signed-off-by: Piotr Sikora <piotrsikora@google.com>

### CODE REVIEW

**Comment by PiotrSikora** [comment]
Cherry-picking to 1.4 to future-proof it against cherry-picks in the future.

File: scripts/check-style.sh [hunk 1]
```diff
@@ -18,9 +18,10 @@
...
 
-CLANG_VERSION_REQUIRED="8.0.0"
-CLANG_FORMAT=$(which clang-format-${CLANG_VERSION_REQUIRED%%.*})
-if [[ ! -x "${CLANG_FORMAT}" ]]; then
+CLANG_VERSION_REQUIRED="9.0.0"
+CLANG_FORMAT=$(which clang-format)
+CLANG_VERSION="$(${CLANG_FORMAT} -version 2>/dev/null | cut -d ' ' -f 3 | cut -d '-' -f 1)"
+if [[ ! -x "${CLANG_FORMAT}" || "${CLANG_VERSION}" != "${CLANG_VERSION_REQUIRED}" ]]; then
   # Install required clang version to a folder and cache it.
   CLANG_DIRECTORY="${HOME}/clang"
```
Comment (lambdai) [inline scripts/check-style.sh]: Hmm, I didn't realize it was using $HOME/clang

**Pull request merged**